code/src/Components/ThoughtsList.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react'
+import moment from 'moment'
+import "./thoughtslist.css"
+
+export const ThoughtsList = () => {
+  const THOUGHTS_URL = "https://technigo-thoughts.herokuapp.com/"
+  const [thoughts, setThoughts] = useState([])
+
+  useEffect(() => {
+    //Ask the server for the thoughts using a GET request
+    fetch(THOUGHTS_URL)
+      .then((res) => {
+        // GET the JSON of the response body. All info is retreived here.
+        return res.json()
+      })
+      .then(data => {
+        // Set the state based on the response
+        setThoughts(data)
+      })
+  }, [])
+
+  const onLiked = (thoughtId) => {
+    const updatedThoughts = thoughts.map(thought => {
+      if (thought._id === thoughtId) {
+        thought.hearts += 1
+      }
+      return thought
+    })
+    setThoughts(updatedThoughts)
+  }
+
+  const handleClick = (thoughtId) => {
```

Comment:
In case you didn't know - you don't need `()` around if you only passing one argument
```suggestion
  const handleClick = thoughtId => {
```